lib/camunda-cloud/CleanUpTimerExpressionBehavior.js

```diff
@@ -0,0 +1,72 @@
+import {
+  is
+} from 'bpmn-js/lib/util/ModelUtil';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import {
+  getTimerEventDefinition,
+  isTypeSupported
+} from './util/TimerUtil';
+
+
+/**
+ * Zeebe BPMN behavior ensuring that bpmn:BusinessRuleTask only has one of the following:
+ *
+ * (1) zeebe:CalledDecision
+ * (2) zeebe:TaskDefinition and zeebe:TaskHeaders
+ */
+export default class CleanUpBusinessRuleTaskBehavior extends CommandInterceptor {
```

Comment:
```suggestion
export default class CleanUpTimerExpressionBehavior extends CommandInterceptor {
```

+ Update documentation on what it does.

Reply:
🤦 on me